plugins/plugins-available/business_process/root/bp-2.20-1.js

```diff
@@ -916,6 +916,8 @@ function bp_update_status(evt, node) {
         if(looks_like_regex(service)) {
             var filter = service.replace(/^(w|b):/, '');
             link = "<a href='status.cgi?style=detail&dfl_s0_type=host&dfl_s0_op=%3D&dfl_s0_value="+host+"&dfl_s0_type=service&dfl_s0_op=%7E&dfl_s0_value="+filter+"&backend="+bp_backend+"'><img src='"+url_prefix+"themes/"+theme+"/images/command.png' border='0' alt='Goto Service Details' title='Goto Service Details' width='16' height='16'><\/a>";
+        } else if (host == service) {
```

Comment:
What about `} else if (n.bp_ref && host == service) {`
Having a service named like a host is unlikely but not limited to business processes.

Reply:
Your version is better since mine also breaks the link corresponding to the BP parent node because its bp_ref is null.  Now I believe the ```host == service``` isn't needed anymore.  How about just:
 ```} else if (n.bp_ref) {```?

Reply:
you are right, sounds good.